<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(86, 184, 139);
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>iRODS basic GUI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabBrowser">
       <attribute name="title">
        <string>Browser</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QPushButton" name="UploadButton">
            <property name="text">
             <string>Up/Download</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>iRODS path: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="federationButton">
            <property name="text">
             <string>Federation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inputPath">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 87, 83);</string>
            </property>
            <property name="text">
             <string>/zoneName/home/user</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="homeButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="collTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 87, 83);</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Checksum</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="viewTabs">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 87, 83);</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="preview">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <attribute name="title">
            <string>Preview</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTextBrowser" name="previewBrowser"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="metadata">
           <attribute name="title">
            <string>Metadata</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTableWidget" name="metadataTable">
              <property name="minimumSize">
               <size>
                <width>600</width>
                <height>0</height>
               </size>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <column>
               <property name="text">
                <string>Key</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Value</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Units</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="6" column="2">
               <widget class="QLineEdit" name="metaUnitsField"/>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Value</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLineEdit" name="metaKeyField"/>
              </item>
              <item row="7" column="0">
               <widget class="QPushButton" name="metaAddButton">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Units</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Key</string>
                </property>
               </widget>
              </item>
              <item row="7" column="2">
               <widget class="QPushButton" name="metaDeleteButton">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="metaValueField"/>
              </item>
              <item row="8" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="7" column="1">
               <widget class="QPushButton" name="metaUpdateButton">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="accession">
           <attribute name="title">
            <string>Permissions</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTableWidget" name="aclTable">
              <property name="minimumSize">
               <size>
                <width>600</width>
                <height>0</height>
               </size>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <column>
               <property name="text">
                <string>User name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Zone</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Permission type</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="4" column="2">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Permission type</string>
                </property>
               </widget>
              </item>
              <item row="7" column="3">
               <widget class="QPushButton" name="aclAddButton">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_11">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Recursive</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QComboBox" name="aclBox">
                <item>
                 <property name="text">
                  <string>----</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>null</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>read</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>write</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>own</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="8" column="2">
               <widget class="QLabel" name="aclError">
                <property name="styleSheet">
                 <string notr="true">color: rgb(164, 0, 0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLineEdit" name="aclUserField"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>User name</string>
                </property>
               </widget>
              </item>
              <item row="6" column="3">
               <widget class="QComboBox" name="recurseBox">
                <item>
                 <property name="text">
                  <string>False</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>True</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="8" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="aclZoneField"/>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>UserZone</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="resources">
           <attribute name="title">
            <string>Resources</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableWidget" name="resourceTable">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <column>
               <property name="text">
                <string>Resource name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabUpload">
       <attribute name="title">
        <string>Up and Download</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCloseSession"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCloseSession">
   <property name="text">
    <string>Close Session</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
